tests/UpgradeAaveV2ConfiguratorPayload.t.sol

```diff
@@ -30,12 +32,12 @@ contract ProposalPayloadProofOfReserveTest is Test {
   }
 
   function testExecuteProposal() public {
-    // deploy all contracts
-    Deploy script = new Deploy();
-    script.deployContracts();
+    ConfiguratorMock configurator = new ConfiguratorMock();
```

Comment:
Why using a mock here?

Reply:
v2 uses 0.6.12 solc and it makes a lot of pain to combine 0.8.* and 0.6.* in the same file